docs/sphinx/snippets/cpp/using/backends/dynamics.cpp

```diff
@@ -0,0 +1,236 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2025 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+// Compile and run with:
+// ```
+// $nvq++ --target dynamics dynamics.cpp -o dynamics && ./dynamics
+// ```
+
+#include "cudaq/algorithms/evolve.h"
+#include "cudaq/dynamics_integrators.h"
+#include "cudaq/operators.h"
+#include <cmath>
+#include <complex>
+#include <cudaq.h>
+#include <iostream>
+#include <vector>
+
+using namespace cudaq;
+
+int main() {
+  // [Begin Transmon]
+  // Parameters
+  double omega_z = 6.5;
+  double omega_x = 4.0;
+  double omega_d = 0.5;
+
+  // Qubit Hamiltonian
+  auto hamiltonian = sum_op<product_op>(0.5 * omega_z * spin_op::z(0));
```

Comment:
This is also not a correct type. The below should work (please confirm by running the example):
```suggestion
  auto hamiltonian = spin_op(0.5 * omega_z * spin_op::z(0));
```